static/images/backpack-sm.svg

```diff
@@ -0,0 +1,43 @@
+<svg id="backpack-sm" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 49.7 49.7">
```

Comment:
these svgs could probably also get hidden from aria/focus?

Reply:
@mirisuzanne I think these are already getting `aria-hidden="true"`?

![Screen Shot 2021-03-29 at 10 52 03 AM](https://user-images.githubusercontent.com/552316/112855466-d2af9980-907c-11eb-826f-d21aafc555e3.png)

Reply:
oh, ok - not sure how I missed that.